recipes/libffi/0001-libffi-Don-t-be-smart-about-toolexeclibdir.patch

```diff
@@ -1,17 +1,17 @@
-From 9d4196a8868abd07dfb1ce9a3fb1dda23872061e Mon Sep 17 00:00:00 2001
+From 21b2c114d58f7e305a4c26bf761dd2496bdcf18e Mon Sep 17 00:00:00 2001
 From: Edward Hervey <bilboed@bilboed.com>
 Date: Thu, 29 Jan 2015 15:37:02 +0100
 Subject: [PATCH] libffi: Don't be smart about toolexeclibdir
```

Comment:
might as well leave it as is and not apply it? commented out in the recipe with a note about 3.2